src/covid19sim/simulator.py

```diff
@@ -1176,8 +1176,8 @@ def at(self, location, city, duration):
         location.add_human(self)
         self.wear_mask()
 
-        self.leaving_time = duration + self.env.now
-        self.start_time = self.env.now
+        self.start_time   = self.env.now
+        self.leaving_time = self.start_time + duration*SECONDS_PER_MINUTE*TICK_MINUTE
```

Comment:
Since duration is in minutes, it should be divided by `TICK_MINUTE` rather than multiplied by it to be in the time scale as `self.start_time`

Reply:
@satyaog See Slack. This is precisely the critical problem I am referencing above, and it was precisely my intention to nuke `TICK_MINUTE`; But as noted above I have not yet stacked that fix atop this branch because it (obviously) breaks bit-exact reproducibility against the original (broken) code. I use bit-exact reproducibility to verify that my changes to timescales, which should formally never affect disease spread, indeed did not do so.